jobs/furnishings/src/furnishings/stage_processors/stage_one.py

```diff
@@ -319,8 +390,7 @@ async def process(app: Flask, qsm: QueueService):  # pylint: disable=redefined-o
         for batch_processing in batch_processings:
             await processor.process(batch_processing)
         processor.generate_paper_letters()
-        # TODO: send AR and transition pdf to BCMail+
-        # TODO: mark MAIL entries as PROCESSED/FAILED
```

Comment:
Missing the step to update the status of successfully uploaded entries to `PROCESSED`. And you need to update the unit test for it too.

Reply:
For this, I remember you created a PR a while back that put things in QUEUED if I recall correctly. Currently, if the SFTP process is processed properly, the entries are still in QUEUED. Do we still want to change that to PROCESSED?

Reply:
Yes, that's what I mean. I put things in `QUEUED`, waiting for SFTP implemetation completed. If the SFTP process is processed correctly, the entries are changed to `PROCESSED`. If not, it should be changed to `FAILED`.

Reply:
Ahh, I see. Makes a lot of sense. Thanks for catching that, will fix it.

Reply:
Updated, when the SFTP process is a success:
![sftp success](https://github.com/user-attachments/assets/552ead84-fc3a-4d8e-8f25-e219e8281e81)